docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3.4'
+
+networks:
+  streams:
+    external:
+      name: 'streams.reaction.localhost'
+
+services:
+
+  grafana:
+    image: 'grafana/grafana:5.4.3'
+    networks:
+      streams:
+    ports:
+      - '3001:3000'
```

Comment:
Is there a general way or best practice for mapping internal/external ports? Hardcoding 3001 instead of 3000 because we know there's already something else on 3000 is a pretty fragile workflow, but I'm not aware of anything better.

Reply:
I guess there's a way but it requires planning and coordination and therefore depends heavily on your organization's ability or inability for that kind of thing. But in my personal projects I would allocate a block on contiguous ports and use those. For example, on a phone keypad "REA" is 732 so we could start at 7320 and go up from there.